src/radical/pilot/session.py

```diff
@@ -850,6 +850,13 @@ def _get_pilot_sandbox(self, pilot):
         return pilot_sandbox
 
 
+    def _get_pilot_fs(self, pilot):
+        resource = pilot.description.get('resource')
```

Comment:
This might be simpler as:
```py
def _get_pilot_fs(self, pilot):
    fs_url = ru.Url(self._get_resource_sandbox())
    fs_url.path = '/'
    return fs_url
```
That would also benefit from the caching we have for the resource sandbox - what do you think?

Reply:
I did walk through the code, and got to know the caching part, but this doesn't work on my machine. 
Probably because it's the resource sandbox URL which I am not targeting. 
I want eg. three.ssh.filesystem_endpoint config from resource config. 

This code is giving me `sftp:///home/workstation/radical.pilot.sandbox` which is sandbox URL and not remote fs endpoint.

Moreover, do we want to pick eg. `three.local.filesystem_endpoint` instead? 
I picked eg. `three.ssh.filesystem_endpoint config` from resource config.

Reply:
YES this works much better. I saw the _cache logic and changed as per what you commented.